app/assets/stylesheets/application.scss

```diff
@@ -54,3 +54,11 @@ $govuk-use-legacy-palette: false;
     }
   }
 }
+
+.article-container {
+  margin-bottom: govuk-spacing(4);
```

Comment:
our components have a default `govuk-spacing(6)`, unless there's a strong reason for using another spacing scale I would use the default one.